src/DevCenter/DevCenter/ChangeLog.md

```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
 
+## Version 3.0.0
```

Comment:
- In the changelog, please use verbs in the past tense.
- Please remove version 3.0.0, we will bump the version you automatically